### DISCUSSION TITLE
OpenRGB

### DISCUSSION THREAD

**Discussion by dursonATWIT** [elrumo/macOS_Big_Sur_icons_replacements#1295] [Icon request] [answered]
Hello, I use OpenRGB to control my laptop keyboard lighting, but the icon looks really out of place. 
<img width="512" alt="OpenRGB" src="https://user-images.githubusercontent.com/71397056/136258921-d3feccb0-e703-4731-b0b0-a569a3d28ae4.png">
(Sorry for the poor quality, It's the best I could find)
Thank you!

**Comment by ChilliChompa** [comment, marked as answer]
Hi, I made 3 for you to get started. I have just reused what I found online so not the highest quality. Enjoy.

![OpenRGB](https://user-images.githubusercontent.com/7188187/136348723-5971062d-2190-4db4-b4a0-da7142ba39fc.png)

![OpenRGB Alt](https://user-images.githubusercontent.com/7188187/136348718-0c9f252c-1454-4ee2-a978-28e25c703567.png)
![OpenRGB Alt 2](https://user-images.githubusercontent.com/7188187/136348696-49731560-4317-4b82-82e3-9b3876dfba0a.png)
